plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestingSqlServer.java

```diff
@@ -35,16 +38,14 @@ public TestingSqlServer()
 
     public TestingSqlServer(boolean snapshotIsolationEnabled)
     {
-        super(DockerImageName.parse("microsoft/mssql-server-linux:2017-CU13").asCompatibleSubstituteFor("mcr.microsoft.com/mssql/server:2017-CU12"));
+        container.addEnv("ACCEPT_EULA", "yes");
```

Comment:
You have part of containier init as field initialized, and part here.
Consolidate in one place.